packages/ruby/spec/readme/har/response_serializer_spec.rb

```diff
@@ -1,5 +1,6 @@
 require "readme/har/response_serializer"
 require "readme/filter"
+require "http_response"
```

Comment:
I don't think we need this? We should be using the double created by the `build_response` helper so we can test the behavior of the serializer in isolation without also needing to deal with logic in the response parser.